Formula/rgbds.rb

```diff
@@ -22,17 +22,28 @@ class Rgbds < Formula
   depends_on "pkg-config" => :build
```

Comment:
```suggestion
  depends_on "pkg-config" => :build
  depends_on "rust" => :build
```
We forgot something 🙂 (squash this again please, thanks)

Reply:
Done, I was looking at another example (`ripgrep` in this case) to be sure of the correct dependency, and noticed that we don't install the provided Zsh completions, which I've also fixed.